README.md

```diff
@@ -30,6 +30,13 @@ To enable setting of several properties in SonarQube on project level is require
 
 The property "projectKey" or "userSlug" are mandatory in order to decide which API endpoint should be used.
 
+## Serving images for PR decoration
+By default, images for PR decoration are served as static resources on the SonarQube server as a part of Community Branch Plugin. 
+
+If you use a SonarQube server behind a firewall and/or PR service (Github, Gitlab etc..) hasn't access to SonarQube server, you should change `Images base URL` property in `General > Pull Request` settings.
+
+It is used to load the images for the PR decoration. [Default public base images URL](https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/master/src/main/resources/static).
```

Comment:
This line seems confusing. I suspect you're trying to say that `Anyone needing to set this value can use the URL 'https://raw.githubusercontent.com/mc1arke/sonarqube-community-branch-plugin/master/src/main/resources/static', or download the files from this location and host them themself`.

I also have a slight concern about pointing people to master, as I've then either got to keep any old images on master (and therefore included in each plugin release), or risk breaking rendering of user's pull requests if the images in this plugin ever change. I'd potentially ask users to load images off the tag related to their release, but don't know if that over-complicates the setup.

Reply:
@mc1arke Thanks for review, I've updated README according your recommendation.

Regarding your doubts about pointing to master, unfortunately I see no easy and universal solution to handle possible breaking changing for images hosting without increasing complexity and/or decreasing of flexibility.

\# | Solution | Possible issues | Comment
---|-----------|-----------------|------------
1 | Use static resources | **No issues** | Images are always actual
2 | Input field for URL |  | Manual input
2.1 | Github URL pointing to master | Risk of breaking rendering | Maintainer should keep all previous version of images in master to provide backward compatibility
2.2 | Github URL pointing to tag | Risk of breaking rendering | User should keep this link in-sync with installed version of plugin
2.3 | Self-hosted images  | Risk of breaking rendering | User should keep them in-sync with installed version of plugin
3 | Additional config switch, like `Use official public images URL: yes/no` | **No issues** | Plugin will generate a proper images URL using actual release tag by itself (not clear how to make it works for unreleased snapshots and PRs)

1+2 most simple and pretty flexible
1+3 most reliable but not flexible
1+2+3 flexible but complex from my prospective

Reply:
@mc1arke Could you please provide any suggestion about this?

Reply:
Sorry for the delayed response. I'm happy to stick with your changes as-is for now.